src/react-sortable.tsx

```diff
@@ -49,11 +49,14 @@ export class ReactSortable<T extends ItemInterface> extends Component<
     this.ref = createRef<HTMLElement>();
 
     // make all state false because we can't change sortable unless a mouse gesture is made.
-    const newList = props.list.map((item) => ({
-      ...item,
```

Comment:
Spread operator will create a JS object, and the class information will be broken, using Object.assign will preserve the original object, and add the required properties